### DISCUSSION TITLE
Camera gotchas

### DISCUSSION THREAD

**Discussion by szupi-ipuzs** [make-all/tuya-local#696] [Q&A]
So I am trying to integrate my birdfeeder and its main feature is a camera. I found one device in tuya-local that has camera as primary entity (bcom_intercom_camera.yaml) and tried to create a similar config - the camera entity showed up in HA, but was not usable and was showing nothing (I will attach screenshots later).
Are the cameras supported in general? Maybe I need to do some extra steps to make it work?

I have a valid local key - that's for sure, as the other entities I created (eg. a siren switch) were working fine.

**Comment by make-all** [comment]
Currently the camera support only has support for still image capture, as the one camera that is supported only offered still images locally.  If your camera does not have the exact same format, it probably won't work without further changes to the camera implementation.

**Comment by szupi-ipuzs** [comment]
That explains it. Now that you mention it, I realized my camera also does not offer video feed locally. As for the still images, I need to dig deeper into dpids to see if there are any related to this feature.
So, I reckon, since this is tuya-**local** integration, you're not planning to support fetching video feed from tuya servers?
